templates/common/_base/files/usr-local-sbin-set-valid-hostname.yaml

```diff
@@ -0,0 +1,83 @@
+mode: 0755
```

Comment:
need filesystem :)

Reply:
Does it? The other templates lack it. Digging around in the other templates, most -- if not all -- lack it too.